webrtc.html

```diff
@@ -8852,6 +8854,34 @@ <h2 id="sec.stats-model">Statistics Model</h2>
       [[!WEBRTC-STATS]].</p>
     </section>
     <section>
+      <h3>The stats selection algorithm</h3>
+      <p>
+	The <dfn>stats selection algorithm</dfn> is as follows:
+	<ul>
+	  <li>If the selector is a MediaStreamTrack, return the following objects:
```

Comment:
Including objects "referencing track stats" is backwards, and does end up including everything. The goal here IMHO should be to return an object without broken references. So I vote for recursive *and* "referenced by", but not "referencing". Makes sense?